--- rpihddevice-1.0.0/ilclient/Makefile	2015-11-06 21:27:08.000000000 +0100
+++ vdr-rpihddevice-1.0.0/ilclient/Makefile	2015-11-06 22:29:21.584697788 +0100
@@ -3,7 +3,17 @@
 
 CFLAGS+=-DSTANDALONE -D__STDC_CONSTANT_MACROS -D__STDC_LIMIT_MACROS -DTARGET_POSIX -D_LINUX -fPIC -DPIC -D_REENTRANT -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -U_FORTIFY_SOURCE -Wall -g -DHAVE_LIBOPENMAX=2 -DOMX -DOMX_SKIP64BIT -ftree-vectorize -pipe -DUSE_EXTERNAL_OMX -DHAVE_LIBBCM_HOST -DUSE_EXTERNAL_LIBBCM_HOST -DUSE_VCHIQ_ARM -Wno-psabi
 
-INCLUDES+=-I$(SDKSTAGE)/opt/vc/include/ -I$(SDKSTAGE)/opt/vc/include/interface/vcos/pthreads -I$(SDKSTAGE)/opt/vc/include/interface/vmcs_host/linux
+#INCLUDES += -I$(VDRSRC)/include -I$(ILCDIR) 
+#INCLUDES += -I$(SYSROOT)/usr/include -I$(SYSROOT)/usr/include/interface/vcos -I$(SYSROOT)/usr/include/interface/vcos/pthreads
+#INCLUDES += -I$(SYSROOT)/usr/include/interface/vmcs_host/linux $(OPTVC)/include
+
+INCLUDES += -I$(VDRSRC)/include -I$(ILCDIR) -I$(OPTVC)/include -I$(OPTVC)/include/interface/vcos/pthreads -I$(OPTVC)/include/interface/vmcs_host/linux
+
+LDLIBS  += -lbcm_host -lvcos -lvchiq_arm -lopenmaxil -lGLESv2 -lEGL -lpthread -lrt
+LDLIBS  += -Wl,--whole-archive $(ILCDIR)/libilclient.a -Wl,--no-whole-archive
+LDFLAGS += -L$(OPTVC)/lib -Wl,-O1
+
+#INCLUDES+=-I$(SDKSTAGE)/opt/vc/include/ -I$(SDKSTAGE)/opt/vc/include/interface/vcos/pthreads -I$(SDKSTAGE)/opt/vc/include/interface/vmcs_host/linux
 
 all: $(LIB)
 
